doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst

```diff
@@ -77,15 +77,19 @@ Programming the sample
 **********************
 
 The :ref:`multicore_hello_world` sample is a multicore sample running on both the Application core (``cpuapp``) and the Peripheral Processor (PPR, ``cpuppr``).
-It uses the ``nrf54h20dk_nrf54h20_cpuapp@soc1`` build target.
+It uses the ``nrf54h20pdk_nrf54h20_cpuapp`` build target.
 
 To build and program the sample to the nRF54H20 PDK, complete the following steps:
 
 1. Connect the nRF54H20 PDK to you computer using the IMCU USB port on the PDK.
 #. Navigate to the :file:`nrf/samples/multicore/hello_world` folder containing the sample.
-#. Build the sample by running the following command::
+#. Build the sample for application and radio cores by running the following command::
 
-      west build -b nrf54h20dk_nrf54h20_cpuapp@soc1
+      west build -p -b nrf54h20pdk_nrf54h20_cpuapp -T sample.multicore.hello_world.nrf54h20pdk_cpuapp_cpurad .
```

Comment:
```suggestion
      west build -p -b nrf54h20pdk_nrf54h20_cpuapp -T sample.multicore.hello_world.nrf54h20pdk_cpuapp_cpurad
```

Reply:
this doesn't work, sample path needs to be specified when using -T